### ISSUE TITLE
batch_normalization should be involved in Test mode

### ISSUE THREAD

**Issue by liurida** [tobegit3hub/tensorflow_template_application#17]
When is_train, the bn is added. But test, it's ignored. based on the doc in the page, https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py
logic should be like the desc below:
    is_training: Whether or not the layer is in training mode. In training mode
      it would accumulate the statistics of the moments into `moving_mean` and
      `moving_variance` using an exponential moving average with the given
      `decay`. When it is not in training mode then it would use the values of
      the `moving_mean` and the `moving_variance`.
